extensions/panache/hibernate-orm-panache/deployment/pom.xml

```diff
@@ -72,6 +72,12 @@
             <artifactId>rest-assured</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.platform</groupId>
+            <artifactId>junit-platform-launcher</artifactId>
+            <version>1.6.1</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Where is this dependency used? I wonder if we should add to `quarkus-junit5-internal` instead?

Reply:
Also fixing the version is a bad idea, the dependency is already declared in the `junit-bom` that is imported in the runtime BOM.

Reply:
I used `.setExpectedException(BuildException.class)` instead of the JUnit platform Launcher so this is no longuer needed.